lua/flutter-tools/lsp/rename.lua

```diff
@@ -0,0 +1,142 @@
+local M = {}
+
+local lazy = require("flutter-tools.lazy")
+local lsp_utils = lazy.require("flutter-tools.lsp.utils") ---@module "flutter-tools.lsp.utils"
+local path = lazy.require("flutter-tools.utils.path") ---@module "flutter-tools.utils.path"
+local ui = lazy.require("flutter-tools.ui") ---@module "flutter-tools.ui"
+
+local api = vim.api
+local util = vim.lsp.util
+local lsp = vim.lsp
+local fn = vim.fn
+
+--- Computes a filename for a given class name (convert from PascalCase to  snake_case).
+local function file_name_for_class_name(class_name)
+  local starts_uppercase = class_name:find("^%u")
+  if not starts_uppercase then return nil end
```

Comment:
```suggestion
  if not starts_uppercase then return end
```
No point in the explicit `nil` IMO